src/sentry/lang/native/minidump.py

```diff
@@ -137,8 +139,29 @@ def merge_symbolicator_minidump_response(data, response):
     if response.get('crashed') is not None:
         data['level'] = 'fatal' if response['crashed'] else 'info'
 
-    if response.get('timestamp'):
-        data['timestamp'] = float(response['timestamp'])
+    # If we set a too old timestamp then this affects event retention and
+    # search. XXX(markus): We should figure out if we could run normalization
+    # after event processing again. Right now we duplicate code between here
+    # and event normalization
```

Comment:
_nit_: You could make it more explicit by moving this logic somewhere into EventManager. There's a second normalization in Python right now (user agents), that would qualify for this.

Reply:
Done